server/test_common/testGQLRequests.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+/**
+ * Mock requests used to test the gql elements of the server. Query names referenced in feature files, E.g `When I make a 'mockTopicsQuery' gql request to ...`. These can be generated using GraphiQL and it' export feature
+ */
+const mockTopicsQuery = {
+  query: '{\n  topics {\n    name\n    partitions\n    replicas\n  }\n}\n',
```

Comment:
Does this need to be a string? Can supertest take an object as input?

Reply:
`These can be generated using GraphiQL and it' export feature`

Suspect using graphql tag it could be a typical query definition

Will fix the it' in the next PR